distribution/server/src/assemble/bin.xml

```diff
@@ -23,6 +23,7 @@
           xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.1.0 http://maven.apache.org/xsd/assembly-2.1.0.xsd">
   <id>bin</id>
   <formats>
+    <format>dir</format>
```

Comment:
this looks like only convenient for debugging purposes. please remove.
the Pulsar repo is already huge when you build it (more than 5GB)

Reply:
Yep. The generated folder takes 421M. I will revert it.

Reply:
Reverted. Also, finish `lib/presto` to `trino` migration and test locally with examples. Don't run all integration tests though.